admin-dev/themes/new-theme/js/pages/order/view/order-view-page.js

```diff
@@ -199,6 +199,7 @@ export default class OrderViewPage {
 
     EventEmitter.on(OrderViewEventMap.productListPaginated, (event) => {
       this.orderProductRenderer.paginate(event.numPage);
+      this.orderProductRenderer.toggleColumnLocation();
```

Comment:
This would be more logical to do this in the `orderProductRenderer.paginate` function

Reply:
Fixed